### ISSUE TITLE
[Est:3] [Wordpress & Woocommerce] Custom post types

### ISSUE THREAD

**Issue by sofia-doofinder** [doofinder/doofinder-woocommerce#339]
Enlace a notion: https://witty-pelican-a6c.notion.site/Wordpress-Woocommerce-Custom-post-types-fcfef57bb51f452da03a5354547812aa?pvs=4

**Comment by davidmolinacano** [comment]
After talking to @IgnacioPursalsZ we have decided to discern between posts and products. Posts will include posts, pages and very custom post type.

**Comment by davidmolinacano** [comment]
I forgot to put the technical validation here:

**Validated with @eduardogomez97  ✅** 

Although the task itself is very straightforward, I discussed with him the way of showing the metafields for posts, especifically if we should have a dropdown or not. After having discussed it, we decided to discuss this with the Product team directly. But the approach of replicating the logic used for products is validated.
